content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -138,6 +138,279 @@ All the resources related to VM on the KubeVirt cluster will be created in a ded
 
 ---
 
+### Advanced configuration
+
+#### Scheduling settings
+It's possible to control how the tenant nodes are scheduled on the infrastructure nodes.
+![Scheduling](./Scheduling.png)
+
+We provide 3 different types of scheduling for the KubeVirt tenant nodes:
+- Ensure co-location on the same infrastructure node (*Pod Affinity Preset*).
+- Prevent co-locotation on the same infrastructure node (*Pod Anti Affinity Preset*).
+- Schedule on nodes having some specific labels (*Node Affinity Preset*).
+
+This setup is done in the `Initial Nodes` step of KKP dashboard when creating a cluster.
+
+![Dashboard](./Dashboard-scheduling.png) 
```

Comment:
The UI for Affinity Settings has been updated in https://github.com/kubermatic/dashboard/pull/4813 so you might want to update this screenshot once that is merged. 

Here's how it looks now:

![screenshot-localhost_8000-2022 08 11-22_21_30](https://user-images.githubusercontent.com/13975988/184194974-86449e5e-f270-4c44-a184-914c853f6202.png)

Reply:
Screenshot is updated.